test/runnable/test34.d

```diff
@@ -697,18 +697,18 @@ void test34()
 
 void foo35()
 {
-        uint a;
-        uint b;
-        uint c;
-        extern (Windows) int function(int i, int j, int k) xxx;
-
-        a = 1;
-        b = 2;
-        c = 3;
-
-        xxx = cast(typeof(xxx))(a + b);
-        asm { int 3; }
-        xxx( 4, 5, 6 );
+    uint a;
+    uint b;
+    uint c;
+    extern (Windows) int function(int i, int j, int k) xxx;
+
+    a = 1;
+    b = 2;
+    c = 3;
+
+    xxx = cast(typeof(xxx))(a + b);
+    assert(0);
+    xxx( 4, 5, 6 );
```

Comment:
```
runnable/test34.d(711): Error: null dereference in function _D6test345foo35FZv
```
@WalterBright - Care to explain?  It's not a null dereference, but the address would be 0x3 - however this line of code is unreachable.

I assume the error didn't occur as its assumed that xxx may change inside the asm block?  That's trivial to synthesize without assembly.